great_expectations/__init__.py

```diff
@@ -16,6 +19,7 @@
 # Both of the following import paths will trigger this file, causing the registration to occur:
 #   import great_expectations as gx
 #   from great_expectations.core import ExpectationSuite, ExpectationConfiguration
+register_core_metrics()
```

Comment:
Can we time the startup cost before and after this?

Reply:
`hyperfine` shows they are microseconds off